tests/CollateralRiskSteward.t.sol

```diff
@@ -0,0 +1,449 @@
+
+    currentEmodeConfig = AaveV3Ethereum.POOL.getEModeCategoryCollateralConfig(eModeId);
+
+    eModeCategoryUpdates[0] = IEngine.EModeCategoryUpdate({
+      eModeCategory: eModeId,
+      ltv: currentEmodeConfig.ltv - 50, // 0.5% absolute increase
+      liqThreshold: currentEmodeConfig.liquidationThreshold - 10, // 0.1% absolute increase
+      liqBonus: (currentEmodeConfig.liquidationBonus - 100_00) - 50, // 0.5% absolute increase
+      label: EngineFlags.KEEP_CURRENT_STRING
+    });
+    // expect revert as UpdateNotInRange
+    vm.expectRevert(IRiskSteward.UpdateNotInRange.selector);
+    steward.updateEModeCategories(eModeCategoryUpdates);
+
+    afterEmodeConfig = AaveV3Ethereum.POOL.getEModeCategoryCollateralConfig(eModeId);
+    afterLabel = AaveV3Ethereum.POOL.getEModeCategoryLabel(eModeId);
+
+    assertEq(afterEmodeConfig.ltv, currentEmodeConfig.ltv);
+    assertEq(afterEmodeConfig.liquidationThreshold, currentEmodeConfig.liquidationThreshold);
+    assertEq(afterEmodeConfig.liquidationBonus, currentEmodeConfig.liquidationBonus);
+    assertEq(afterLabel, label);
+
+    lastUpdated = steward.getEModeTimelock(eModeId);
+
+    assertEq(lastUpdated.eModeLtvLastUpdated, 0);
+    assertEq(lastUpdated.eModeLiquidationThresholdLastUpdated, 0);
+    assertEq(lastUpdated.eModeLiquidationBonusLastUpdated, 0);
+  }
+
+  function test_updateEModeCategories_outOfRange() public virtual override {
+    vm.warp(MIN_DELAY);
+    super.test_updateEModeCategories_outOfRange();
+  }
+
+  function test_updateEModeCategories_debounceNotRespected() public virtual override {
+    uint8 eModeId = 1;
+    DataTypes.CollateralConfig memory currentEmodeConfig = AaveV3Ethereum
+      .POOL
+      .getEModeCategoryCollateralConfig(eModeId);
+
+    IEngine.EModeCategoryUpdate[] memory eModeCategoryUpdates = new IEngine.EModeCategoryUpdate[](
+      1
+    );
+    eModeCategoryUpdates[0] = IEngine.EModeCategoryUpdate({
+      eModeCategory: eModeId,
+      ltv: currentEmodeConfig.ltv + 50, // 0.5% absolute increase
+      liqThreshold: currentEmodeConfig.liquidationThreshold + 10, // 0.1% absolute increase
+      liqBonus: (currentEmodeConfig.liquidationBonus - 100_00) + 50, // 0.5% absolute increase
+      label: EngineFlags.KEEP_CURRENT_STRING
+    });
+
+    vm.startPrank(riskCouncil);
+    // expect revert as minimum time has not passed for next update
+    vm.expectRevert(IRiskSteward.DebounceNotRespected.selector);
+    steward.updateEModeCategories(eModeCategoryUpdates);
+
+    vm.warp(MIN_DELAY - 1);
+
+    // expect revert as minimum time has not passed for next update
+    vm.expectRevert(IRiskSteward.DebounceNotRespected.selector);
+    steward.updateEModeCategories(eModeCategoryUpdates);
+  }
+
+  function test_updateEModeCategories_sameUpdate() public virtual override {
+    vm.warp(MIN_DELAY);
+    super.test_updateEModeCategories_sameUpdate();
+  }
+
+  /* ----------------------------- Collateral Tests ----------------------------- */
```

Comment:
where are the positive tests?

Reply:
separately, can you check is not a problem when using _max relative change_? i think the only place where we are using is on `debtCeiling`, but please confirm that.
main concern is that if value is 0, we may not be able to set to non-zero even with max_uint128 value as `maxPercentChange`